instrumentation/aws_sdk/README.md

```diff
@@ -32,6 +32,30 @@ OpenTelemetry::SDK.configure do |c|
   c.use_all
 end
 ```
+### Configuration options
```

Comment:
I love these updated configuration docs! We've also put similar content in the `instrumentation.rb` file for the library. Would you mind adding them there too? 

Here's an example for how we did it with Sidekiq: https://github.com/open-telemetry/opentelemetry-ruby-contrib/blob/main/instrumentation/sidekiq/lib/opentelemetry/instrumentation/sidekiq/instrumentation.rb

Reply:
Happy to!